apps/module-presale/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@ablack/module-presale",
+  "version": "0.1.0",
+  "author": "Aragon Black <contact@aragon.black>",
+  "contributors": [
+    "Alejandro Santander <palebluedot@gmail.com>"
+  ],
+  "repository": "github:AragonBlack/fundraising",
+  "homepage": "https://github.com/AragonBlack/fundraising",
+  "bugs": "https://github.com/AragonBlack/fundraising/issues",
+  "license": "(GPL-3.0-or-later OR MIT)",
```

Comment:
Yep. @ajsantander you can update that one. I copy-pasted Aragon's default licensing but it may have changed in the meanwhile. Gonna update all the other repos too.